akka-http-tests/src/test/scala/akka/http/scaladsl/CustomMediaTypesSpec.scala

```diff
@@ -42,7 +42,7 @@ class CustomMediaTypesSpec extends AkkaSpec with ScalaFutures
       val routes = extractRequest { r ⇒
         complete(r.entity.contentType.toString + " = " + r.entity.contentType.getClass)
       }
-      val binding = Http().bindAndHandle(routes, host, port, settings = serverSettings)
+      val _ = Http().bindAndHandle(routes, host, port, settings = serverSettings)
```

Comment:
We either keep it or don't val it at all, I'd say?

Reply:
You are right, no need to keep the `val`. I was playing with some search-and-replace expressions :)